requirements/compiled.txt

```diff
@@ -5,7 +8,7 @@ Pillow==2.3.0
 
 # For JWT to sign App receipts:
 # Temporary.
-M2Crypto==0.20.2
+M2Crypto==0.22.3
```

Comment:
mmmm, yeah.  That's what was available as a .whl so I upgraded it.  The tests pass - we can consider this just upgrading, right?  (So it will be QA'd and pushed as usual, since I changed the requirements file)